conf/application.conf

```diff
@@ -168,7 +168,7 @@ tracking-consent-frontend {
  gtm.container = "b"
 }
 
-accessibility-statement.service-path = "/report-and-manage-your-allowance-for-customs-duty-waiver-claims"
+accessibility-statement.service-path = "http://localhost:12346/accessibility-statement/report-and-manage-your-allowance-for-customs-duty-waiver-claims"
```

Comment:
This is absolute here, but in https://github.com/hmrc/app-config-base/pull/9089/files#diff-c5386e670e14fc6c48f297865d134ae363f7de33b86cf9f38606b5e1e59c6e73R80 it was relative.